fog-opennebula.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
     s.add_dependency 'fog-json',  '~> 1.1'
     s.add_dependency 'fog-xml',   '~> 0.1'
     s.add_dependency 'nokogiri',  '< 1.13' if RUBY_VERSION < '2.6.0'
-    s.add_dependency 'opennebula'
+    s.add_dependency 'opennebula', '~> 5.8', '>= 5.8.2'
```

Comment:
OpenNebula 5.8 is no longer supported. This gem is tested actively against the latest OpenNebula release, which is 6.6 at the moment. There shouldn't be opennebula version constraints at all in the gemspec file.

Reply:
Thank you! I deleted these changes